.github/runner_all_enrt.py

```diff
@@ -0,0 +1,67 @@
+import inspect
+import logging
+
+from lnst.Controller import Controller
+from lnst.Controller.ContainerPoolManager import ContainerPoolManager
+from lnst.Controller.MachineMapper import ContainerMapper
+from lnst.Recipes.ENRT.BaseEnrtRecipe import BaseEnrtRecipe
+
+import lnst.Recipes.ENRT as enrt_recipes
+
+podman_uri = "unix:///run/podman/podman.sock"
+image_name = "lnst"
+ctl = Controller(
+    poolMgr=ContainerPoolManager,
+    mapper=ContainerMapper,
+    podman_uri=podman_uri,
+    image=image_name,
+    debug=1,
+    network_plugin="custom_lnst"
+)
+
+params_base = dict(
+    perf_tests=['tcp_stream'],
+    perf_duration=5,
+    perf_iterations=2,
+    perf_warmup_duration=0,
+    ping_count=1,
+    perf_test_simulation=True,
+)
```

Comment:
You may want to check if we want to override the `offload_combinations` parameter as the default contains multiple settings which we probably don't need.

Reply:
for sure, that's also part of the problems for the recipe runs as veth nics don't support offloads and the commands fail..